src/storage/storage.rs

```diff
@@ -16,12 +23,12 @@ impl Storage {
         client_options.app_name = Some("Guesser Game Api".to_string());
         let client = Client::with_options(client_options)?;
         let db = client.database("guesser-game-api");
-        let secret_word_collection = db.collection::<SecretWord>("secret");
+        let secret_collection = db.collection::<SecretEntity>("secret");
```

Comment:
Since a collection is about storing multiple values, let's change this collection name to be `secrets`

```suggestion
        let secret_collection = db.collection::<SecretEntity>("secrets");
```